.github/workflows/build_and_test.yml

```diff
@@ -1,6 +1,18 @@
 name: Build and test
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - "master"
```

Comment:
We want to run the tests on other branches as well before creating the pull request.

Reply:
For the PR from forked repo, if we only run the tests when the PR is opened, we won't be able to test the changes when new changes are pushed to the PR.
<img width="388" alt="image" src="https://github.com/mage-ai/mage-ai/assets/80284865/bb7b9e40-5101-4240-9dd3-5888f30c5373">

Reply:
ok, I have removed everything except the paths.